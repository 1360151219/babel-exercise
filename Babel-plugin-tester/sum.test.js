const pluginTester = require('babel-plugin-tester').default
const identifierReversePlugin = require('./plugin.js')

pluginTester({
    plugin: identifierReversePlugin,
    tests: {
        'case1:': 'const a=1;', // 输入输出都是同个字符串
        'case2:': { // 指定输入输出的字符串
            code: 'const a=1;',
            output: 'const hh = 1;',
        },
        'case3:xxxxxx': { // 指定输入字符串，输出到快照文件中，对比测试
        code: `
        const a = 1;
      `,
            snapshot: true,
        },
    }
})