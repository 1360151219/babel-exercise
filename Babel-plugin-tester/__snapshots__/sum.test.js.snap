// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unknown plugin case3:xxxxxx: case3:xxxxxx 1`] = `

const a = 1;

      ↓ ↓ ↓ ↓ ↓ ↓

const hh = 1;


`;
